mkpr

```diff
@@ -84,10 +84,11 @@ if [ $# == 0 ]; then
 elif [ $# == 2 ] && [ "$1" == "checkout" ] && [[ "$2" =~ ^[0-9]+$ ]]; then
     checkout_with_pr_number "$2"
 elif [ $# == 2 ] && [ "$1" == "push" ] && [[ "$2" =~ ^[0-9]+$ ]]; then
```

Comment:
Not convinced that this works, since it's checking that we have exactly two argumnets.